packages/fast-components-react-msft/src/background/background.tsx

```diff
@@ -76,9 +76,12 @@ export default class Background extends Foundation<
                 designSystem={this.getDesignSystemOverrides(color)}
                 designSystemMergingFunction={this.props.designSystemMergingFunction}
             >
-                <this.tag {...this.unhandledProps()} style={style}>
-                    {this.props.children}
-                </this.tag>
+                {(this.tag && (
+                    <this.tag {...this.unhandledProps()} style={style}>
```

Comment:
Can you change the `BackgroundHandledProps` interface to accept null as a valid value also? Otherwise I don't think projects with strict null checks will be able to use this feature. Also, can you add a note in the prop interface that when tag is `null` no element will be rendered?